src/planwise/endpoint/scenarios.clj

```diff
@@ -30,7 +30,7 @@
          (not-found {:error "Scenario not found"})
          (response (scenarios/get-scenario-for-project service scenario project)))))
 
-   (GET "/:id/csv" [id :as request]
+   (GET "/:id/:name" [id :as request]
      (response (scenarios/export-providers-data service (Integer. id))))
```

Comment:
Set a download filename by setting the [`Content-Disposition`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition) HTTP [header](http://ring-clojure.github.io/ring/ring.util.response.html#var-header).